packages/garbo/src/familiar/freeFightFamiliar.ts

```diff
@@ -68,6 +72,25 @@ export function menu(options: MenuOptions = {}): GeneralFamiliar[] {
       });
     }
 
+    if (
+      gooseDroneEligible() &&
+      get("gooseDronesRemaining") < copyTargetCount()
+    ) {
+      familiarMenu.push({
+        familiar: $familiar`Grey Goose`,
+        expectedValue:
+          // It takes 9 experience to go from level 5 to 6 and emit a drone
+          clamp(
+            getModifier("Familiar Experience") / 9,
+            0,
+            // The limit to how valuable any emission can be is how many drones are actually gonna hit the copyTarget
+            copyTargetCount() - get("gooseDronesRemaining"),
+          ) * valueDrops(globalOptions.target),
+        leprechaunMultiplier: 0,
+        limit: "experience",
+      });
+    }
```

Comment:
Put this behind a check on `mode` or else it'll unnecessarily run on barf constantly

Reply:
Do we call freeFightFamiliar for barf? I was under the impression the whole codeblock was build to mode: "free":

`const DEFAULT_MENU_OPTIONS = {
  canChooseMacro: true,
  location: $location`none`,
  extraFamiliars: [],
  includeExperienceFamiliars: true,
  allowAttackFamiliars: true,
  mode: "free",`